pkg/dnfjson/dnfjson.go

```diff
@@ -693,6 +725,9 @@ type depsolveResult struct {
 
 	// (optional) contains the solver used, e.g. "dnf5"
 	Solver string `json:"solver"`
+
+	// (optional) contains the SBOM for the depsolved transaction
+	Sbom json.RawMessage `json:"sbom"`
```

Comment:
should this also be a pointer and omitempty as it's optional?
```suggestion
	Sbom *json.RawMessage `json:"sbom,omitempty"`
```

Reply:
IMHO no need to make this a pointer, because it is a `[]byte`. `Solver` is also optional and it is not omitempty. While I agree that adding `omitempty` should be there, it has no actual impact on the behavior, because we always only unmarshalling from JSON to the struct.

Reply:
I've added `omitempty` to optional struct members